devspace.yaml

```diff
@@ -12,11 +15,13 @@ images:
     createPullSecret: false
     dockerfile: ./Dockerfile.devspace
     rebuildStrategy: ignoreContextChanges
-    docker: {}
+    docker:
+      args:
+      - --platform=linux/${GOARCH}
 
 deployments:
   azure-creds-secret:
-    namespace: validator-plugin-azure-system
+    namespace: default
```

Comment:
curious why you're putting the secret in the default namespace?

Reply:
I found that I had to do this to get devspace working again after I integrated the changes in the main branch. It was just after the Azure auth changed. I'll take a look at this again later to see if we really need to change this here.